integration_tests/integration/barcode/selectEnvelopeSize.spec.ts

```diff
@@ -1,40 +1,22 @@
 import Page from '../../pages/page'
-import FindRecipientByPrisonNumberPage from '../../pages/barcode/findRecipientByPrisonNumber'
-import CreateNewContactPage from '../../pages/barcode/createNewContact'
-import ReviewRecipientsPage from '../../pages/barcode/reviewRecipients'
-import ChooseBarcodeOptionPage from '../../pages/barcode/chooseBarcodeOption'
 import SelectEnvelopeSizePage from '../../pages/barcode/selectEnvelopeSize'
-import PrintCoversheetsPage from '../../pages/barcode/printCoversheets'
 
 context('Select envelope size', () => {
+  let selectEnvelopeSizePage: SelectEnvelopeSizePage
+
   beforeEach(() => {
-    cy.task('reset')
-    cy.task('stubAuthToken')
-    cy.task('stubRequestLink')
-    cy.task('stubVerifyLink')
-    cy.visit('/link/verify-link?secret=a-valid-secret')
-    cy.task('stubGetPrisonRegister')
-    cy.visit('/barcode/find-recipient')
-    Page.verifyOnPage(FindRecipientByPrisonNumberPage).submitWithValidPrisonNumber()
-    Page.verifyOnPage(CreateNewContactPage).submitWithValidValues()
-    Page.verifyOnPage(ReviewRecipientsPage).prepareBarcodes()
-    Page.verifyOnPage(ChooseBarcodeOptionPage).continueToCoversheet()
+    selectEnvelopeSizePage = SelectEnvelopeSizePage.goToPage()
   })
 
   it('should redisplay form with errors given no envelope size selected', () => {
-    const selectEnvelopeSizePage = Page.verifyOnPage(SelectEnvelopeSizePage)
-
     selectEnvelopeSizePage.submitWithoutSelectingEnvelopeSize()
 
     selectEnvelopeSizePage.hasErrorContaining('Select an option')
-    selectEnvelopeSizePage.signOut().click()
```

Comment:
These sign outs were not required... as suspected Cypress handles the sessions for us.